02.ComputeShader/main.cpp

```diff
@@ -5,13 +5,18 @@
 
 using namespace nbl;
 
-class ComputeShaderSampleApp : public ApplicationBase
+// TODO better way of handling this
```

Comment:
Agreed this shit looks usgly, specially since it's the first thing you see in the example and Examples should inherit from hit.